book/src/developing.md

```diff
@@ -35,3 +35,7 @@ If your driver supports additional features, CSI "capabilities" can be used to a
 
 This is an partial list, please see the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md) for a complete list of capabilities.
 Also see the [Features](features.md) section to understand how a feature integrates with Kubernetes.
+
+## Remain Informed
```

Comment:
Maybe this should go first at the beginning? It can be easily skimmed if it's at the end

Reply:
Sure